integration_test/ops_agent_test.go

```diff
@@ -1893,6 +1901,153 @@ func TestPrometheusMetrics(t *testing.T) {
 		}
 	})
 }
+
+// Test the Counter and Gauge metric types using a JSON Prometheus exporter
+// The JSON exporter will connect to a Python http server that serve static JSON files
+func TestPrometheusMetricsWithJSONExporter(t *testing.T) {
+	t.Parallel()
+	gce.RunForEachPlatform(t, func(t *testing.T, platform string) {
+		t.Parallel()
+		if gce.IsWindows(platform) {
+			t.SkipNow()
+		}
+		ctx, logger, vm := agents.CommonSetup(t, platform)
+
+		type fileToUpload struct {
+			local, remote string
+		}
+
+		filesToUpload := []fileToUpload{
+			{local: "testdata/data.json",
+				remote: "data.json"},
+			{local: "testdata/json_exporter_config.yaml",
+				remote: "json_exporter_config.yaml"},
+		}
+
+		workDir := path.Join(workDirForPlatform(vm.Platform), "json_exporter")
+
+		for _, file := range filesToUpload {
+			f, err := os.Open(file.local)
+			if err != nil {
+				t.Fatal(err)
+			}
+			err = gce.UploadContent(ctx, logger, vm, f, path.Join(workDir, file.remote))
+			if err != nil {
+				t.Fatal(err)
+			}
+		}
+
+		packages := []string{"wget", "python3"}
+		err := agents.InstallPackages(ctx, logger.ToMainLog(), vm, packages)
+		if err != nil {
+			t.Fatalf("failed to install %v with err: %s", packages, err)
+		}
+
+		// Install either tmux or screen to run the JSON exporter and the http server in the background
+		tmux_err := agents.InstallPackages(ctx, logger.ToMainLog(), vm, []string{"tmux"})
```

Comment:
Did we need `tmux` to run things in the background? I was fairly certain there is a bash way to run things in the background too no? Like appending `&` at the end or something?

Worth looking into at the very least to simplify the dependencies of this test.

Reply:
Good point! I was using tmux/screen when developing the test because I can easily verify the two processes are running. Let me try `nohup` with `&` now.